src/asim/configs/airport.CBX/InternalExternalDestinationChoiceAlternatives.csv

```diff
@@ -1,13 +1,13 @@
 alt,taz,name,mgraOut,mgraRet,region,iePct,CBX.Pct,SAN.Pct
-1,1,San Ysidro,6867,6634,1,0.0669,0.000,0.000
-2,2,Otay Mesa,7066,7066,1,0.0169,0.000,0.000
-3,3,East Otay Mesa,7123,7123,1,0,0.000,0.000
-4,4,Tecate,21895,21895,1,0.0053,0.000,0.000
-5,5,Jacumba,22094,22094,1,0,0.000,0.000
-6,6,I-8,21470,21470,2,0.0463,0.025,0.025
-7,7,SR78,21461,21461,2,0.0047,0.027,0.027
-8,8,SR79,20824,20824,3,0.0064,0.008,0.008
-9,9,Pala Road,16215,16215,3,0.0096,0.009,0.009
-10,10,I-15,15889,15889,3,0.3985,0.125,0.125
-11,11,Foothill Tollway,15301,15301,4,0,0.000,0.000
-12,12,I-5,15301,15301,4,0.4453,0.806,0.806
+1,1,San Ysidro,24064,24064,1,0.0669,0,0
+2,2,Otay Mesa,9388,9388,1,0.0169,0,0
```

Comment:
@JoeJimFlood Otay Mesa POE is 9387 for the crossborder model (can be seen here: https://github.com/SANDAG/ABM/blob/0d633c6d56a8961726f7a88f0a676f408af47278/src/asim/configs/crossborder/preprocessing.yaml#L45). Looking at an ArcGIS Pro Map of MGRA15, it also _looks_ like it's 9387. 
Unsure of what's correct, or if both are.